.github/workflows/tests.yml.j2

```diff
@@ -24,29 +30,18 @@ jobs:
         # * This is still a matrix because we might re-enable ELN one day and then we will need
         #   a matrix here.
         #}
-        release: ['', 'branched_fedora']
+        release: [{% for rel in releases_for_branch %}{% if not loop.first %}, {% endif %}{% if rel == current_release_entry %}''{% else %}'{$ rel.release $}'{% endif %}{% endfor %}]
```

Comment:
Can you please ideally decompose this into multiple lines & comments explaining the steps ? I have hard time wrapping my head around it like this on a single line & reasoning about how and what the result will look like. :)

Reply:
Indeed it was not very readable. Moved the logic above into the variables, where statements are in multiple lines and also now it can be reused.